vnext/ReactUWP/Views/ShadowNodeBase.cpp

```diff
@@ -60,9 +72,15 @@ void ShadowNodeBase::onDropViewInstance()
 
 void ShadowNodeBase::ReplaceView(XamlView view)
 {
+
   SetTag(view, GetTag(m_view));
 
   m_view = view;
+
+  // Force unhook from old view
+  UpdateKeyboardEventHooks(nullptr);
```

Comment:
>nullptr [](start = 27, length = 7)

This looks like a bug - nullptr is ignored below.

Reply:
It's on purpose. the new implementation doesn't have this. and it will directly call hook/unhook